modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -111,6 +110,58 @@ public void contribute(OpenAPI openAPI, UriInfo uriInfo) {
 		}
 	}
 
+	private void _addRelationshipPaths(
```

Comment:
Good luck with this. 
I prefer this way, but Brian changed my first PR to avoid methods that only be used once 🤷‍♂️ 
Let's try again, I think is more clear this way.

Reply:
I thought the same while I was doing it, but let's see what happens. I hope this as my little friday lottery